ros/src/inertial_sense_ros.cpp

```diff
@@ -1959,9 +2089,24 @@ void InertialSenseROS::GPS_geph_callback(eDataIDs DID, const geph_t *const msg)
     geph.dtaun = msg->dtaun;
     switch (DID)
     { 
-    case DID_GPS1_RAW:      rs_.gps1_raw.pubGEp.publish(geph);       break;
-    case DID_GPS2_RAW:      rs_.gps2_raw.pubGEp.publish(geph);       break;
-    case DID_GPS_BASE_RAW:  rs_.gpsbase_raw.pubGEp.publish(geph);   break;
+    case DID_GPS1_RAW:
+    if (rs_.gps1_raw.pubGEp)
```

Comment:
These "if" should be indented under the case, to match the indent level of the following "break" (indented 1 depth further than the enclosing braces of the 'switch' statement.